templates/helm-values.tpl.yaml

```diff
@@ -53,22 +53,37 @@ kube-prometheus:
         group_wait: 30s
         group_interval: 5m
         repeat_interval: 12h
-        receiver: 'opsgenie'
+        receiver: slack-customer
         routes:
+        - match_re:
+            namespace: ^(?:infrastructure|kube-system)$
+          routes:
+          - match:
+              severity: warning
+            receiver: slack-runtime
+            routes:
+            - match:
+                group: persistence
+              receiver: slack-persistence
+          - match:
+              severity: critical
+            receiver: opsgenie
         - match:
-            severity: warning
-          receiver: slack-runtime
+            alertname: TargetDown
```

Comment:
Couple of questions:
- `TargetDown` is always meant for us?
- Is there any severity other than `warning` and `critical`? If so, that will get routed to the customer slack, no?

Reply:
- TargetDown is a special case. it can be our targets or customer, but it has no namespace label to select it. thats why I sent it to us
- warning and critical are the only ones configured as far as I know